### PR TITLE
ENT-933 Add spectator and participant profiles of rigorousMock

### PR DESCRIPTION
**Pull request by andr3ej**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -52,7 +52,6 @@ buildscript {
     ext.typesafe_config_version = constants.getProperty("typesafeConfigVersion")
     ext.fileupload_version = '1.3.3'
     ext.junit_version = '4.12'
-    ext.mockito_version = '2.10.0'
```
Comment (mnesbit) [inline build.gradle]: I do actually like having version variables, so not that keen on losing this. I know we probably have only one use site, but I keep finding myself having to re-declare dependencies in other configurations at some point.

Comment (andr3ej) [reply build.gradle]: whoever redeclares should add the variable

**Comment by mnesbit** [comment]
Seems to fail some IRS test

**Comment by andr3ej** [comment]
all green now if anyone feels like approving

**Pull request merged**